### DISCUSSION TITLE
Licensing

### DISCUSSION THREAD

**Discussion by rockerbacon** [rockerbacon/ice-cream-machine-core#14] [Technical]
I added the GPLv3 license to the project without too much thinking. It would be a better idea to discuss this a little bit before we have a large volume of code.

**Comment by rockerbacon** [comment]
I'm fond of copyleft licenses. They might throw off the stiff gaming corporations but I see this as a project for gamers above anything else. Besides, we have a highly decoupled design that would allow anyone to integrate the software into proprietary systems without "infecting" them.

After looking at the [FSF licensing guide](https://www.gnu.org/licenses/license-recommendations.html.en), I'm more inclined to use [AGPL](https://www.gnu.org/licenses/why-affero-gpl.html.en) over GPL. Our software could potentially be used in future cloud gaming servers or something similar and it could derail into something completely proprietary under these scenarios.

**Reply by nlaranjeira01** [reply]
My name is Nelson Antunes Laranjeira Neto and I approve this message.

**Reply by rockerbacon** [reply]
Case closed, then

**Comment by rockerbacon** [comment]
## Summary of the final decision

We will re-license from GPLv3 to AGPLv3 with the goal of keeping the software free and further protecting it on remote usage scenarios.
